app.js

```diff
@@ -79,12 +78,15 @@ Test this function by hand in the console to get it working, and when you think
 var testArray = [2,3,4]; //eslint-disable-line
 
 function sumArray(testArray){ //eslint-disable-line
-
+  var sumArray = [];
+  sumArray[0] = testArray[0] + testArray[1] + testArray[2];
```

Comment:
Using `+` arithmetic operator is not allowed!

What if `testArray` has more or fewer than three elements?